test/silicon_upf.jl

```diff
@@ -0,0 +1,62 @@
+include("run_scf_and_compare.jl")
+include("testcases.jl")
+
+function run_silicon_pbe(T ;Ecut=5, grid_size=15, spin_polarization=:none, kwargs...)
+    # These values were computed using ABINIT with the same kpoints as testcases.jl
```

Comment:
Abinit has UPF support ... I did not know that.

Reply:
My fault here😅. This is copy paste from `test/silicon_pbe.jl`, I didn't run the calculation and update the reference. But Abinit __do support__ UPF `v1` which is parsed by `UPF.jl` as well. I will ask @azadoks about how to run it and maybe can use the result as a reference here.